CHANGELOG.md

```diff
@@ -150,6 +150,7 @@ You can find its changes [documented below](#070---2021-01-01).
 - Update docs of `RawLabel`: does not require `ArcStr`([#1886] by [@Maan2003])
 - Fix `Controller` links for `Click` ([#2158] by [@yrns])
 - Delete inaccurate line for `KeyEvent` ([#2247] by [@amtep])
+- Added examples in `TextBox` ([#2284] by [@ThomasMcAndrew])
```

Comment:
I'm actually not sure if github markdown's links are case sensitive. But probably "andrew" is better just to be safe...

Reply:
@jneem  Good catch, Thanks!